vulture_os/portal/settings.py

```diff
@@ -114,10 +114,9 @@
 LANGUAGE_CODE = 'en-us'
 TIME_ZONE = 'UTC'
 USE_I18N = True
-USE_L10N = True
```

Comment:
Even if enabled by default, it's better to keep explicitely setting the parameter

Reply:
It is deprecated and throw a warning on django check routine : https://github.com/adamchainz/django-upgrade#use_l10n